From 756f19b828e98a4344645cfbfd7fa8ea85734ddc Mon Sep 17 00:00:00 2001
From: Mika Laitio <lamikr@pilppa.org>
Date: Fri, 3 May 2024 14:24:53 -0700
Subject: [PATCH 2/3] ubuntu search fix for ROCM SMI library

CMake search behaves differently in ubuntu
compared for example to fedora or mageia and
does not find the library from lib64 folder
unless explicitly asked to search from there.

Signed-off-by: Mika Laitio <lamikr@pilppa.org>
---
 CMakeLists.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3ab06343..e465f140 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -129,7 +129,8 @@ find_path(ROCM_SMI_ROOT "include/rocm_smi/rocm_smi.h"
     )
 
 find_library(ROCM_SMI_LIBRARY rocm_smi64
-    PATHS "${ROCM_SMI_ROOT}/lib")
+    PATHS "${ROCM_SMI_ROOT}/lib"
+          "${ROCM_SMI_ROOT}/lib64")
 
 add_library(rocwmma INTERFACE)
 target_link_libraries(rocwmma INTERFACE hip::device hip::host OpenMP::OpenMP_CXX ${ROCM_SMI_LIBRARY})
-- 
2.41.3

